scripts/configure-debian

```diff
@@ -12,7 +12,7 @@ cd $BUILD
 cmake	-DSWIG_EXECUTABLE=/usr/bin/swig3.0 \
 	-DPLUGINS="ALL" \
 	-DTOOLS="ALL" \
-	-DBINDINGS="ALL" \
+	-DBINDINGS="STABLE" \
```

Comment:
Maybe ALL is not so bad here (even if it really means ALL).

For the default, we do not have to provide "-DBINDINGS" at all.